miio/toiletlid.py

```diff
@@ -37,6 +45,11 @@ def work_state(self) -> int:
         """Device state code"""
         return self.data["work_state"]
 
+    @property
+    def work_mode(self) -> str:
+        """Device working mode"""
+        return ToiletlidOperatingMode((self.work_state - 1) // 16).name
```

Comment:
Should this be returning rather the `ToiletLidOperatingMode` instead of t he string? Also, should the work_state be adapted already in the `__init__`?